Ginger/GingerCoreCommon/VariablesLib/VariableList.cs

```diff
@@ -50,7 +50,14 @@ public override string VariableUIType
 
         public override string GetFormula()
         {
```

Comment:
please explain why changes were needed in VariableList class

Reply:
Because we are doing ResetValue() in the constructor. And the values of Formula and ValueList are coming null as the first time, as the constructor is being called before the activities are loaded. So in ValueList class there is just the condition of Null Value check to avoid the NullValueException.